src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,18 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
```

Comment:
Consider moving the import statements to the top of the file for better readability and consistency. 📚